gamemodes/terrortown/gamemode/client/cl_shop.lua

```diff
@@ -5,6 +5,50 @@
 
 shop = shop or {}
 
+shop.favorites = shop.favorites or {}
+shop.favorites.databaseName = "ttt2_shop_favorites"
+shop.favorites.savingKeys = {}
+
+sql.CreateSqlTable(shop.favorites.databaseName, shop.favorites.savingKeys)
+
+shop.favorites.orm = orm.Make(shop.favorites.databaseName)
```

Comment:
Not sure how to handle this now. It actually needs an initialization after migration. In all other cases that should work.
Maybe only migration I reset the orm.

Reply:
i'm not exactly sure what you mean here.
As long as the migration has run before this gets called everything should be alright, or am i missing something?